pallets/token-manager/src/lib.rs

```diff
@@ -1092,4 +1092,15 @@ impl<T: Config> BridgeInterfaceNotification for Pallet<T> {
 
         Ok(())
     }
+
+    fn on_incoming_event_processed(event: &EthEvent) -> DispatchResult {
+        match &event.event_data {
+            EventData::LogLifted(d) => return Self::process_lift(event, d),
+            EventData::LogAvtGrowthLifted(d) => return Self::process_avt_growth_lift(event, d),
```

Comment:
Would it be "safer" to add these once we add the filters to the runtime and remove them from eth-events?

Reply:
can do, but the filter is expected to define and constrain the event types the runtime process. The pallets should just be processing events as they come through the interface.